modules/core/src/main/scala/sangria/execution/Executor.scala

```diff
@@ -43,7 +44,9 @@ case class Executor[Ctx, Root](
         userContext,
         exceptionHandler,
         scalarMiddleware,
-        false)(um)
+        false,
+        errorsLimit = queryValidator.errorsLimit
```

Comment:
The idea was to define errorsLimit only once (e.g. on the queryValidator) and re-use in other places